Firestore/Source/Remote/FSTDatastore.mm

```diff
@@ -282,9 +285,9 @@ - (void)lookupDocuments:(NSArray<FSTDocumentKey *> *)keys
                                    [FSTDatastore logHeadersForRPC:rpc RPCName:@"BatchGetDocuments"];
                                    FSTAssert(!response, @"Got response after done.");
                                    NSMutableArray<FSTMaybeDocument *> *docs =
-                                       [NSMutableArray arrayWithCapacity:keys.count];
-                                   for (FSTDocumentKey *key in keys) {
-                                     [docs addObject:results[key]];
+                                       [NSMutableArray arrayWithCapacity:keys.size()];
+                                   for (const DocumentKey &key : keys) {
+                                     [docs addObject:results[(FSTDocumentKey *)key]];
```

Comment:
Same comment that Rich made in the other PR re. C-style cast.

Reply:
done